dolphin/db/sqlalchemy/api.py

```diff
@@ -192,18 +195,19 @@ def _access_info_get(context, storage_id, session=None):
     return result
 
 
-def _access_info_get_query(context, session=None):
-    return model_query(context, models.AccessInfo, session=session)
+def _access_info_get_query(context, model, session=None):
```

Comment:
Why refactored _access_info_get_query? model better not exposed to upper function.

Reply:
i made one function common for all, thats why i have to modify here also